pkg/engine/task/task.go

```diff
@@ -16,14 +16,15 @@ import (
 
 // Context is a engine.task execution context containing k8s client, templates parameters etc.
 type Context struct {
-	Client     client.Client
-	Discovery  discovery.DiscoveryInterface
-	Config     *rest.Config
-	Enhancer   renderer.Enhancer
-	Meta       renderer.Metadata
-	Templates  map[string]string // Raw templates
-	Parameters map[string]string // Instance and OperatorVersion parameters merged
-	Pipes      map[string]string // Pipe artifacts
+	Client      client.Client
+	Discovery   discovery.DiscoveryInterface
+	Config      *rest.Config
+	Enhancer    renderer.Enhancer
+	Meta        renderer.Metadata
+	Templates   map[string]string // Raw templates
+	Parameters  map[string]string // Instance and OperatorVersion parameters merged
+	Pipes       map[string]string // Pipe artifacts
+	ToggleParam string            // Parameter used in ToggleTask
```

Comment:
Is `ToogleParam` a debug leftover (it's already part of `ToggleTask`)

Reply:
yeah, removing it